product-principles/README.md

```diff
@@ -1,6 +1,38 @@
 # Product Principles
 
-As we scale Tilt the company and the product, we will explore how to build a product that best serves our users. These product prinicples help align Tilters and the open source community to build Tilt in a coherent fashion, also allowing for distributed decision making to ensure we can ship quickly. We expect these product principles themselves to evolve over time, so this is always a work in progress.
+As we scale Tilt the company and the product, we will explore how to build a product that best serves our users. These product principles help align Tilters and the open source community to build Tilt in a coherent fashion, also allowing for distributed decision making to ensure we can ship quickly. We expect these product principles themselves to evolve over time, so this is always a work in progress.
 
 ## Usability over documentation
 We believe that Tilt should be usable without reading a separate manual. We strive to design usable interfaces, even for complex features with powerful functionality, without relying on subsequent documentation as a crutch for poor design. We prefer to use our [documentation](https://docs.tilt.dev/) to provide context on broad features, walkthroughs, and API details.
+
+## Infrastructure as Data
+
+One of the core design principles of Kubernetes is "infrastucture as data".
```

Comment:
```suggestion
One of the core design principles of Kubernetes is ["infrastucture as data"](https://twitter.com/kelseyhightower/status/1164194470436302848).
```

@nicks : I've heard of this term before so I'm somewhat certain that Hightower is not the first to coin it (or GitOps). But I think nice in general just to have links to concepts for people to read. Or do you know a better reference off hand?

Reply:
done.

ya, i think it's always hard to pinpoint the exact origin an idea like this - cybernetics dates back to the late 40s, other parts of it are from immutable/functional programming that were popularized by Haskell way back in the 80s/90s, and figuring how it applies to infrastructure is all relatively recent.